cli/src/main/java/com/box/l10n/mojito/cli/command/checks/ContextAndCommentCliChecker.java

```diff
@@ -79,6 +82,14 @@ public CliCheckResult run(List<AssetExtractionDiff> assetExtractionDiffs) {
   private List<ContextAndCommentCliCheckerResult> runChecks(
       List<AssetExtractionDiff> assetExtractionDiffs) {
     return getAddedTextUnitsExcludingInconsistentComments(assetExtractionDiffs).stream()
+        .filter(
+            assetExtractionDiff -> {
```

Comment:
Is this name correct, should it not be `assetExtractorTextUnit`?

Reply:
Done